src/racer/lib.rs

```diff
@@ -11,6 +11,8 @@ extern crate bitflags;
 #[macro_use]
 extern crate derive_more;
 
+extern crate rustc_ap_syntax as syntax;
```

Comment:
Do we really need this?

Reply:
Yes. Without it, building complains about "use of undeclared type or module `syntax`"

Reply:
I suppose the alternative is changing every reference of `syntax` to `rustc_ap_syntax`, but I thought this was a little less intrusive

Reply:
Ah, OK.
Then, how about setting it in toml
```toml
[dependencies.syntax]
package = "rustc-ap-syntax"
version = "606.0.0"
```
like [rustfmt does](https://github.com/rust-lang/rustfmt/pull/3835/files)?

Reply:
Didn't know you could do that, thanks :]

I agree that's much nicer. Changed.